### PR TITLE
Added nft_get_wallets_with_dids RPC to simplify the discovery

### PR DESCRIPTION
**Pull request by paninaro**
of NFT wallets that have an associated DID, and identifying the DID wallet.

It's challenging pulling together a list of DID wallets that have NFTs associated with them. Without this, the GUI needs to invoke several RPCs:

wallets = `get_wallets`
all_dids = for each DID wallet in wallets => `did_get_did`
nft_wallets_with_dids = for each did in all_dids => `nft_get_by_did`
map did wallets to nft_wallets_with_dids

Tested manually using the CLI as well as using the included test code.

### CODE REVIEW

File: chia/rpc/wallet_rpc_api.py [hunk 1]
```diff
@@ -1399,6 +1400,32 @@ async def nft_get_by_did(self, request) -> Dict:
                 return {"wallet_id": wallet.wallet_id, "success": True}
         return {"error": f"Cannot find a NFT wallet DID = {did_id}", "success": False}
 
+    async def nft_get_wallets_with_dids(self, request) -> Dict:
+        assert self.service.wallet_state_manager is not None
+        all_wallets = self.service.wallet_state_manager.wallets.values()
+        did_wallets_by_did_id: Dict[bytes32, uint32] = {
+            wallet.did_info.origin_coin.name(): wallet.id()
+            for wallet in all_wallets
+            if isinstance(wallet, DIDWallet) and wallet.did_info.origin_coin is not None
```
Comment (ytx1991) [inline chia/rpc/wallet_rpc_api.py:1409]: You can use type() to get the wallet type

**Review by ytx1991 — DISMISSED** [review]
overall looks good

**Review by ytx1991 — APPROVED** [review]
lgtm

**Review by trepca — APPROVED** [review]

**Review by wjblanke — COMMENTED** [review]
aok

**Pull request merged**